libs/blocks/quiz/utils.js

```diff
@@ -429,13 +429,21 @@ export const getAnalyticsDataForBtn = (selectedQuestion, selectedCards) => {
   return '';
 };
 
-export const getAnalyticsDataForLocalStorage = (answers) => {
+export const getAnalyticsDataForLocalStorage = (answers, umbrellaProduct, primaryProducts) => {
+  let formattedResultString = '';
   let formattedAnswerString = '';
+  if (umbrellaProduct) {
+    formattedResultString = umbrellaProduct;
+  } else {
+    primaryProducts.forEach((product) => {
+      formattedResultString = formattedResultString === '' ? product : formattedResultString.concat('|', product);
```

Comment:
I was going to suggest that `formattedResultString.concat('|', product)` be turned into 
```
${formattedResultString}|${product}
```
for better readability, but I see this is a pattern used in other places in this file; might still be worth considering

Reply:
Done.